pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/QueryOp.java

```diff
@@ -55,7 +72,97 @@ public void setExpectedResultsFileName(String expectedResultsFileName) {
 
   @Override
   boolean runOp() {
-    System.out.println("Verifying queries in " + _queryFileName + " against results in " + _expectedResultsFileName);
-    return true;
+    System.out.printf("Verifying queries in %s against results in %s\n", _queryFileName, _expectedResultsFileName);
+    try {
+      return verifyQueries();
+    } catch (Exception e) {
+      LOGGER.error("FAILED to verify queries in {}: {}", _queryFileName, e);
+      return false;
+    }
+  }
+
+  boolean verifyQueries()
+      throws Exception {
+    BufferedReader expectedResultReader = null;
+    boolean testPassed = false;
+
+    try (BufferedReader queryReader = new BufferedReader(
+        new InputStreamReader(new FileInputStream(_queryFileName), StandardCharsets.UTF_8))) {
+      if (_queryFileName == null) {
+        LOGGER.error("Result file is missing!");
+        return testPassed;
+      } else {
+        expectedResultReader = new BufferedReader(
```

Comment:
I think you can put this to the same try bracket in  Line 89 as well, so that you don't need to specify the finally block at the end.

Reply:
done